portfolio/src/main/webapp/index.html

```diff
@@ -5,9 +5,11 @@
     <title>Awad Osman</title>
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB59P0iF2uNA3k7SRJb2mDf8aNgrbKpWig&callback=initMap"
+  type="text/javascript"></script>
     <script src="script.js"></script>
   </head>
-  <body onload = "onloader()">
+  <body onload = "onloader(); createMap();">
```

Comment:
Can you call `createMap()` in the `onloader()` function?

Reply:
Did not think of that, thanks for pointing that out!